lib/Backends/CPU/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ if(NOT LLVM_LINK_BIN)
     set(LLVM_LINK_BIN ${LLVM_BINARY_DIR}/bin/llvm-link)
   endif()
 endif()
-if(NOT EXISTS ${LLVM_LINK_BIN})
+if(NOT EXISTS ${LLVM_LINK_BIN} AND NOT MSVC)
```

Comment:
@ayermolo if LLVM_LINK_BIN is missing how would you get 
```
add_custom_command(
    OUTPUT ${GLOW_BINARY_DIR}/CPU/libjit.bc
    COMMAND ${LLVM_LINK_BIN} -o ${GLOW_BINARY_DIR}/CPU/libjit.bc ${CPURuntime_OBJS}
    DEPENDS  ${CPURuntime_OBJS} ${CPURuntime_SRCS}
    WORKING_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}")
```

executed?

Reply:
Well root path for it can be passed through -DCMAKE_PREFIX_PATH. Then when solution is build $(configuration) is resolved. If path is incorrect it results in build time error.
Does that answer your question, or did I miss understand?